modules/gapi/include/opencv2/gapi/imgproc.hpp

```diff
@@ -943,6 +979,73 @@ image type is @ref CV_8UC1.
 GAPI_EXPORTS GMatP NV12toBGRp(const GMat &src_y, const GMat &src_uv);
 
 //! @} gapi_colorconvert
+
+// TODO: move to the separate file modules/gapi/include/opencv2/gapi/video.hpp
+//! @addtogroup gapi_video
+//! @{
+/** @brief Calculates an optical flow for a sparse feature set using the iterative Lucas-Kanade
+method with pyramids.
+
+See @cite Bouguet00 . The function is parallelized with the TBB library.
+
+@note Function textual ID is "org.opencv.video.calcOpticalFlowLK"
```

Comment:
Needs to be fixed once you address the kernel name problem

Reply:
ok